stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatTheme.kt

```diff
@@ -162,6 +172,7 @@ public fun ChatTheme(
     ).onEach { if (it is DownloadPermissionHandler) it.ObservePermissionChanges() },
     attachmentsPickerTabFactories: List<AttachmentsPickerTabFactory> = AttachmentsPickerTabFactories.defaultFactories(),
     videoThumbnailsEnabled: Boolean = true,
+    streamCdnImageResizing: StreamCdnImageResizing = StreamCdnImageResizing(),
```

Comment:
Having this kind of centralized approach was discussed earlier, but there's a drawback to it.

Take the following screenshot into consideration:

![Screenshot_1671549237](https://user-images.githubusercontent.com/37080097/208701577-9f1e73af-7898-41e3-95c9-78a941ef308c.png)

The resized image works great withing the quoted comment, however it's quality is too low for the media preview.

I'm wondering if we should add the ability to resize per component as well, such as adding `StreamCdnImageResizing` to individual factories and UI Components as well.

This would incur a greater development cost, but allow for greater customization. 
However, since this approach would incur a greater BE cost due to potentially resizing a single attachment multiple times, it should not be the default approach, but rather an additional capability. Essentially, the default argument would be the one inside the `ChatTheme`.

Reply:
while it doesn't look great, I think it's okay since the user can always open the attachment to preview something of higher quality. It's common to see lower res images as previews and then you can show them as higher res when previewing details.

Worst case, we can try to make the change if customers request this. We also are not 100% sure how many customers will use the resize, how many use their own CDN etc.

Reply:
Alright, implementing a global way of setting the parameter for now.
Giving fine grained control should be a non-breaking implementation should customers ask for it anyways.